### PR TITLE
Refactoring/tao 5104/remove globals from generis

### PR DESCRIPTION
**Pull request by ionutpad**

### CODE REVIEW

File: actions/class.Authoring.php [hunk 1]
```diff
@@ -43,7 +46,7 @@ public function index()
     	$testService = taoTests_models_classes_TestsService::singleton();
 
     	$class = new core_kernel_classes_Class(CLASS_LTI_TESTCONTENT);
-    	$content = $test->getOnePropertyValue(new core_kernel_classes_Property(TEST_TESTCONTENT_PROP));
+    	$content = $test->getOnePropertyValue(new core_kernel_classes_Property(TestService::PROPERTY_TEST_CONTENT));
```
Comment (zagovorichev) [inline actions/class.Authoring.php:49]: depends on https://github.com/oat-sa/extension-tao-test/pull/188/files#diff-096f34b8841df0fda7fda714e51755f8R37 (v6.5.1)

**Comment by zagovorichev** [comment]
@ionutpad You've forgotten to mention PR dependency
https://github.com/oat-sa/extension-tao-lti/pull/81

**Review by zagovorichev — CHANGES_REQUESTED** [review]

File: actions/class.Authoring.php (hunk 1 above)
Comment (zagovorichev) [reply actions/class.Authoring.php:49]: @ionutpad I'm waiting for update

Comment (ionutpad) [reply actions/class.Authoring.php:49]: Done, Thanks

**Review by zagovorichev — APPROVED** [review]

**Pull request merged**